simln-lib/src/sim_node.rs

```diff
@@ -1067,6 +1058,7 @@ pub async fn ln_node_from_graph(
                 node.1 .0.clone(),
                 graph.clone(),
                 routing_graph.clone(),
+                SystemClock {}
```

Comment:
We should pass this clock in as a parameter to give callers the freedom to choose what impl they want